src/lib/support/Span.h

```diff
@@ -53,6 +59,12 @@ class FixedSpan
 
     const T * data() const { return mDataBuf; }
     size_t size() const { return N; }
+    bool empty() const { return data() == nullptr; }
+    bool data_equal(const FixedSpan & other) const
+    {
+        return (empty() && other.empty()) ||
```

Comment:
Really, people should never use an uninitialized FixedSpan, which is the only way to get `empty()` here....